.github/workflows/base.yml

```diff
@@ -393,21 +393,25 @@ jobs:
         distribution: 'temurin'
         java-version: '11'
     - name: Download GraalVM Maven Repo
+      if: ${{ fromJson(needs.build-vars.outputs.build-from-source) == true && needs.build-vars.outputs.distribution == 'mandrel' && inputs.builder-image == 'null'}}
```

Comment:
`distribution == graalvm` builds those too and we should download them in that case. So it should be "mandrel or graalvm" whatever the syntax for that is.

Reply:
Perhaps we can just drop the condition on `mandrel`. `builder image == null && build from source == true` should be sufficient?

Reply:
Yes